src/components/apps/listing/Listing.js

```diff
@@ -58,12 +66,14 @@ function Listing({
     const [category, setCategory] = useState(selectedCategory);
 
     const [selected, setSelected] = useState([]);
+    const [selectedApp, setSelectedApp] = useState(null);
     const [lastSelectIndex, setLastSelectIndex] = useState(-1);
 
     const [data, setData] = useState(null);
     const [agaveAuthDialogOpen, setAgaveAuthDialogOpen] = useState(false);
     const [detailsEnabled, setDetailsEnabled] = useState(false);
     const [detailsOpen, setDetailsOpen] = useState(false);
+
     const [detailsApp, setDetailsApp] = useState(null);
```

Comment:
GitHub Actions is showing warnings for this line since `detailsApp` and `setDetailsApp` are no longer used.